design-docs/CLI-Subscriptions.md

```diff
@@ -175,6 +175,49 @@ In the case of invalid options, print an error message (stderr) and set the exit
 code to 2. In the case of an API error response, print the response JSON and 
 set code to 1.
 
+## Request
+
+### Interface
+
+```
+planet subscriptions request [OPTIONS]
+
+Generate a subscriptions request.
+
+This command provides support for building the subscription request JSON used to create or 
+update a subscription. It outputs the subscription request.
+
+Options:
+  --name TEXT                     Subscription name. Does not need to be unique.
+                                  [required]
+  --asset-types TEXT              One or more comma-separated asset types [required]
+  --item-types TEXT               One or more comma-separated item-types [required]
+  --geometry JSON                 GeoJSON of area to deliver subscription to. Use 
+                                  --clip only deliver pixels that match this area.
+                                  Can be a json string, filename, or '-' for stdin.
+  --clip                          Use the geometry GeoJSON as the clip geometry for 
```

Comment:
This is an attempt to 'reuse' the geometry JSON. It might be too restrictive, since it could only use the exact same geometry.  If we wanted more flexibility could add an optional argument, but if no argument is given then it'd use the geometry. 

This likely gets complicated trying to combine it with 'tools' JSON. In orders we just don't allow clip and tools in the same operation. But I think we have some convenience that lets you use both. I think we should do #801, so perhaps there's some way to make it easy to input / reuse the same geometry. Or more simple could be that you create a tool.json that is 'ready' for a subscription and would just reuse the geometry JSON when it is there as the clip. Like it'd make a special 'clip' section that would get filled out later.